recipe/meta.yaml

```diff
@@ -11,38 +11,42 @@ source:
   sha256: 5cac870d6d269cdd71d28def4cae310c609c02aac636c0831176188f16021f44
 
 build:
+  skip: True  # [py<37]
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script:
+    - {{ PYTHON }} -m poetry build
```

Comment:
OK. This requires a full poetry, not core. Let me investigate something before we go that route.